expression/builtin_time_vec_generated.go

```diff
@@ -404,6 +404,10 @@ func (b *builtinAddStringAndStringSig) vecEvalString(input *chunk.Chunk, result
 
 		// calculate
 
+		if !isDuration(arg1) {
+			result.AppendNull() // fixed: false
+			continue
+		}
```

Comment:
The file is generated by go generate in expression/generator. We shouldn't edit it by manual.. Maybe we should modify the template in time_vec.go and run `make gogenerate` to generate the code.

Reply:
that's the way

Reply:
@wshwsh12 PTAL, BTW I believe [`subtime`](https://dev.mysql.com/doc/refman/8.0/en/date-and-time-functions.html#function_subtime) have the same issue, should I fix it in this PR?

Reply:
If the reason is the same, I think we can fix it in this pr.